modules/codegen/src/main/scala/com/twilio/swagger/codegen/SwaggerUtil.scala

```diff
@@ -295,6 +295,8 @@ object SwaggerUtil {
         Target.pure(Resolved(t"Double", None, Default(d).extract[Double].map(Lit.Double(_))))
       case d: DecimalProperty =>
         Target.pure(Resolved(t"BigDecimal", None, None))
+      case u: UntypedProperty =>
+        Target.pure(Resolved(t"io.circe.Json", None, None)) // TODO: o.getProperties
```

Comment:
Does `getProperties` exist on `UntypedProperty`? [UntypedProperty.java](https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-models/src/main/java/io/swagger/models/properties/UntypedProperty.java) suggests it doesn't (which would also make sense, as there's no way for it to know)

Reply:
Additionally, something to note here is that SwaggerUtil needs to eventually be collapsed down into the different ProtocolGenerators, otherwise `io.circe.Json` won't be valid for other JSON libraries (`json4s`, support for Java, etc). Not a concern right now, but this is something that'll need to happen for #36

Reply:
yeah, I figured that the `io.circe.Json` was good for now given that the protocol generator stuff isn't parameterized yet. I hope it's as simple as a find/replace of `io.circe.Json` to `${correctProtocolGenerator}` when that happens.